be/src/exec/json_scanner.h

```diff
@@ -178,7 +178,15 @@ class JsonReader {
     std::vector<std::vector<JsonPath>> _parsed_jsonpaths;
     std::vector<JsonPath> _parsed_json_root;
 
-    rapidjson::Document _origin_json_doc; // origin json document object from parsed json string
+    char _value_buffer[4 * 1024 * 1024];
+    char _parse_buffer[512 * 1024];
```

Comment:
It's better to define a constant for 512*1024

Reply:
> The constant is only used here and once, so it can be directly used?

I don't think so. Because we can hardly get the meaning  `512*1024`, and we can get it from the name of a constant

Reply:
So, if we define a varible named kParserBufferSize, can you get some meaning? I think it can be got from _parse_buffer?